src/Util/JsonLD.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * @file src/Util/JsonLD.php
+ */
+namespace Friendica\Util;
+
+use Friendica\Core\Cache;
+use digitalbazaar\jsonld as DBJsonLD;
```

Comment:
This `use` alias is never used, is there a reason to leave it here?

Reply:
Don't know, I must confess.